src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/movement/ModuleInventoryMove.kt

```diff
@@ -139,7 +140,7 @@ object ModuleInventoryMove : ClientModule("InventoryMove", Category.MOVEMENT) {
         movementKeys[key] = pressed
 
         if (behavior == SAFE && mc.currentScreen is InventoryScreen
-        && InventoryManager.isInventoryOpenServerSide && pressed) {
+        && InventoryManager.isInventoryOpen && pressed) {
```

Comment:
This should be isInventoryOpenServerSide because otherwise it will always try to close it regardless of whether it's actually open serverside.